certora/specs/AccessControlDefaultAdminRules.spec

```diff
@@ -0,0 +1,379 @@
+import "helpers.spec"
+import "methods/IAccessControlDefaultAdminRules.spec"
+import "methods/IAccessControl.spec"
+import "AccessControl.spec"
+
+use rule onlyGrantCanGrant filtered {
+  f -> f.selector != acceptDefaultAdminTransfer().selector
+}
+
+// use rule revokeRoleEffect filtered {
+//   role -> role != DEFAULT_ADMIN_ROLE()
+// }
+
+// use rule renounceRoleEffect filtered {
+//   role -> role != DEFAULT_ADMIN_ROLE()
+// }
+
+/*
+┌─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
+│ Helpers                                                                                                             │
+└─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
+*/
+
+function max_uint48() returns mathint {
+    return (1 << 48) - 1;
+}
+
+// defaultAdmin() returns the zero address when there's no default admin
```

Comment:
This comment feels out of place here. I'd  move it to `defaultAdminConsistency`.